icons/rimac.svg

```diff
@@ -0,0 +1 @@
x="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rimac</title><path d="M7.55 11.492c-.029 0-.04.034-.02.055l.716.784a.074.074 0 00.054.026h.681l-.58-.63c-.01-.014-.004-.04.012-.043l.245-.022c.417-.047.612-.234.612-.494 0-.31-.265-.451-.803-.451H5.921l.187.183a.068.068 0 00.054.025h1.975c.4 0 .533.102.533.286 0 .17-.187.268-.584.272zm2.911.865h.603v-1.64h-.603zm4.806-1.623l-1.118.924a.044.044 0 01-.062 0l-1.121-.924c-.012-.013-.028-.017-.047-.017h-.81l1.768 1.456c.02.017.042.017.062 0l1.34-1.103c.015-.013.034 0 .034.02v1.267h.604v-1.64h-.604a.07.07 0 00-.046.017m2.65.008l-.801.878h.323c.02 0 .04-.01.055-.026l.588-.643a.05.05 0 01.074 0l1.26 1.38a.074.074 0 00.056.026h.677l-1.475-1.615a.068.068 0 00-.055-.025h-.65a.06.06 0 00-.05.025m4.189-.025c-.358 0-.654.042-.865.204a.77.77 0 00-.303.613c0 .478.424.818.942.818h.374a.073.073 0 00.043-.012l.12-.09c.004-.004.004-.008.004-.013a.009.009 0 00-.008-.008c-.48-.01-.872-.247-.872-.643 0-.405.347-.67.884-.67h1.351c.02 0 .039-.008.055-.025l.167-.183h-1.892zm.175 3.254v-.515h.1v.515zm-1.558 0v-.515h.102v.426h.463v.09zm-1.16 0v-.515h.1v.515zm-1.04-.12c0 .095-.046.12-.14.12h-.46v-.515h.457c.105 0 .14.025.14.119v.043c0 .038-.008.072-.05.08.038.009.058.039.058.086v.068zm-.1-.272c0-.034-.012-.038-.047-.038h-.36v.132h.364c.03 0 .042-.01.042-.038zm0 .205c0-.034-.012-.04-.047-.04h-.36v.14h.36c.035 0 .046-.002.046-.036zm-2.05.19c-.104 0-.147-.02-.147-.143v-.24c0-.12.043-.144.148-.144h.37c.105 0 .148.026.148.145v.239c0 .119-.04.144-.148.144zm.418-.4c0-.033-.012-.037-.043-.037h-.374c-.035 0-.042.004-.042.038v.277c0 .034.01.038.042.038h.374c.035 0 .043-.004.043-.038zm-1.694.397v-.4h-.004l-.253.357h-.023l-.253-.357h-.004c0 .025.004.06.004.098v.298h-.101v-.516h.12l.246.354.245-.354h.117v.516zm-2.176.004c-.105 0-.148-.021-.148-.144v-.24c0-.12.043-.144.148-.144h.37c.105 0 .148.026.148.145v.239c0 .119-.04.144-.148.144zm.416-.4c0-.034-.011-.038-.042-.038h-.374c-.035 0-.043.004-.043.038v.277c0 .034.012.038.043.038h.374c.035 0 .042-.004.042-.038zm-1.9.396v-.434h-.28v-.086h.657v.086h-.276v.434zm-1.729.004c-.11 0-.14-.017-.14-.144v-.38h.1v.397c0 .034.012.038.043.038h.34c.034 0 .046-.004.046-.038v-.397h.101v.38c0 .123-.03.144-.14.144zm-1.227-.004l-.062-.11h-.378l-.062.11h-.113l.3-.515h.128l.3.515zm-.26-.422l-.137.235h.296l-.136-.235zm-3.59-4.22a8.206 8.206 0 00-2.274-.33 8.206 8.206 0 00-2.276.33.122.122 0 00-.083.114c0 1.988.72 4.45 2.292 5.536a.11.11 0 00.13 0c1.572-1.086 2.295-3.544 2.292-5.536a.113.113 0 00-.08-.114zm-1.332 3.935c-.247.43-.542.816-.892 1.113a.075.075 0 01-.098 0c-.96-.817-1.518-2.26-1.73-3.654l-.244-.134h1.808c.582 0 .657.195.657.338 0 .142-.075.337-.657.337h-.53a.032.032 0 00-.032.032c0 .009.004.017.012.025l1.773 1.833.19.195zm.323-.663l-1.058-1.093c-.016-.016-.004-.04.016-.045l.208-.024c.437-.05.688-.277.688-.561 0-.33-.338-.512-.9-.512H.53a6.388 6.388 0 01-.044-.54.115.115 0 01.09-.123 7.566 7.566 0 011.766-.215h.031c.578 0 1.18.073 1.766.215a.118.118 0 01.09.122 8.441 8.441 0 01-.605 2.776z"/></svg>
```

Comment:
Before we review this, why did you choose to add this as the logo rather than just the "R"?

Reply:
Hi. Since the logo provided in the pdf file downloaded from the media center page included the "RIMAC AUTOMOBILI", I decided to add it too inside the svg. I can amend it by removing the wordings, leaving the "R".

Reply:
I'm not that familiar with the brand so you might be better to judge this, but there are various places where they use just the "R" and I didn't spot anything saying we can't use that :thinking:

Reply:
Will amend it to "R" only later tonight or as soon as possible... Thanks again for your time to review this...

Reply:
Hi. I have made the amendments. Do tell if anything needs to be updated again. Thanks for reviewing.